pkg/sbom/generator/spdx/spdx.go

```diff
@@ -366,6 +364,8 @@ func (sx *SPDX) imagePackage(opts *options.Options) (p *Package) {
 			"SPDXRef-Package-%s", opts.ImageInfo.ImageDigest,
 		)),
 		Name:             opts.ImageInfo.ImageDigest,
+		Version:          opts.ImageInfo.ImageDigest,
+		Supplier:         "Organization: Chainguard, Inc", // TODO
```

Comment:
Should I pull this out of `/etc/os-release` if present? Isn't there somewhere else we do that?